review-tools/ghmerge

```diff
@@ -59,14 +59,12 @@ curl -s https://api.github.com/repos/openssl/$WHAT/pulls/$PRNUM >/tmp/gh$$
 set -- `python -c '
 from __future__ import print_function
 import json, sys;
-print(str(json.load(sys.stdin)["head"]["label"]).replace(":", " "))' </tmp/gh$$`
+input = json.load(sys.stdin)
+print(str(input["head"]["label"]).replace(":", " "),
+      str(input["head"]["repo"]["ssh_url"]))'        </tmp/gh3531`
```

Comment:
This is probably a leftover from testing? `/tmp/gh3531`

Reply:
oops, yes! Fixed.